tests/features/support/steps.js

```diff
@@ -207,6 +215,59 @@ Then(/the link "([^"]+)" should open in (a new window|the same window)/, async (
   await scope.afterStep(scope);
 });
 
+// FEATURE IN DEVELOPMENT. A FOOL'S ERRAND
+let number_map = {first: 0, second: 1, third: 2, fourth: 3, fifth: 4, sixth: 5, seventh: 6, eighth: 7, ninth: 8, tenth: 9, };
+let specified = '?"?(first|second|third|fourth|fifth|sixth|seventh|eighth|ninth|tenth|"[^"]+)?"?';
+let prohibited_choice_words = `I should not see the options? "([^"]+)" ?(?:in the ${specified} choice list)?`;
+let prohibited_choice_words_regex = new RegExp(prohibited_choice_words);
+Then(prohibited_choice_words_regex, async (prohibited, specifier) => {
```

Comment:
I don't think we're using this `Then` statement in any of the tests we have right now. Can we use it in a test, so we can make sure this function works?

Reply:
I do know that in this repo we're using `Then` a lot, both in defining steps and in scenarios, so I don't think we have to worry about that. From what I've seen, you can also use `Then`, `When`, etc. interchangeably. Even if I define with `Then` here, I can use `When` in the actual scenario if I want.